tests/sqlparser_snowflake.rs

```diff
@@ -143,6 +143,26 @@ fn test_single_table_in_parenthesis_with_alias() {
     );
 }
 
+#[test]
+fn test_array_agg_func() {
+    for (sql, canonical) in [
```

Comment:
@SuperBo the only difference between both is the case, right?

I'd use the `verified_stmt` instead of `one_statement_parses_to` using the same case. You're not testing the case structure, but the ARRAY_AGG parsing. There's no need to have a proof on that.

Reply:
Fixed